velox/substrait/VeloxToSubstraitPlan.cpp

```diff
@@ -213,7 +325,68 @@ ::substrait::Plan& VeloxToSubstraitPlanConvertor::addExtensionFunc(
   extensionFunction->set_function_anchor(3);
   extensionFunction->set_name("divide:i32_i32");
 
+  extensionFunction =
+      substraitPlan->add_extensions()->mutable_extension_function();
+  extensionFunction->set_extension_uri_reference(0);
+  extensionFunction->set_function_anchor(4);
+  extensionFunction->set_name("count:opt_i32");
+
+  extensionFunction =
+      substraitPlan->add_extensions()->mutable_extension_function();
+  extensionFunction->set_extension_uri_reference(0);
+  extensionFunction->set_function_anchor(5);
+  extensionFunction->set_name("sum:opt_i32");
+
+  extensionFunction =
+      substraitPlan->add_extensions()->mutable_extension_function();
+  extensionFunction->set_extension_uri_reference(0);
+  extensionFunction->set_function_anchor(6);
+  extensionFunction->set_name("modulus:i32_i32");
+
+  extensionFunction =
+      substraitPlan->add_extensions()->mutable_extension_function();
+  extensionFunction->set_extension_uri_reference(0);
+  extensionFunction->set_function_anchor(7);
+  extensionFunction->set_name("equal:i64_i64");
+
   return *substraitPlan;
 }
 
+const ::substrait::AggregateFunction&
+VeloxToSubstraitPlanConvertor::setAggregateSteps(
```

Comment:
It would be easier to read the code if this method was replaced with a function in an anonymous namespace that takes an core::AggregationNode::Step and returns the corresponding Substrait's phase.

```
aggregateFunction->set_phase(toAggregationPhase(aggregateNode->step()));
```